projects/collection-editor-library/src/lib/components/manage-collaborator/manage-collaborator.component.ts

```diff
@@ -0,0 +1,265 @@
+      console.log(error);
+    });
+    modal.deny();
+  }
+
+  /* to be called when user change tab to add collaborator */
+  getAllusers() {
+    this.isAddCollaboratorTab = true;
+  }
+
+  /* to be called when user change tab to manage collaborator */
+  getCollaborators() {
+    this.isAddCollaboratorTab = false;
+    this.manageCollaboratorTabVisit += 1;
+    if (this.manageCollaboratorTabVisit <= 1) {
+      // tslint:disable-next-line:no-string-literal
+      this.userSearchBody.request.filters['userId'] = this.contentCollaborators;
+      if (!_.isEmpty(this.contentCollaborators)) {
+        this.helperService.getAllUser(this.userSearchBody).subscribe((response: any) => {
+          const count = _.get(response, 'result.response.count');
+          if (!count) {
+            this.collaborators = [];
+          } else {
+            if (_.has(response, 'result.response.content')) {
+              this.collaborators = _.get(response, 'result.response.content');
+            }
+          }
+        },
+        (error) => {
+          this.collaborators = [];
+          this.toasterService.error(_.get(this.configService, 'labelConfig.messages.error.001'));
+          console.log(error);
+        });
+      }
+    }
+  }
+
+  sortUsersList(value) {
+    if (value === 'firstName') {
+      this.users = _.orderBy(this.users, [user => user[value].toLowerCase()]);
+    } else {
+      this.users = _.orderBy(this.users, [user => user.organisations[0].orgName ? user.organisations[0].orgName.toLowerCase() : '']);
+    }
+  }
+
+  searchByKeyword() {
+    const searchRequest = this.userSearchBody;
+    if (this.validateEmail(this.searchKeyword)) {
+      // tslint:disable-next-line:no-string-literal
+      searchRequest.request.filters['email'] = this.searchKeyword;
+    } else if (/^\d+$/.test(this.searchKeyword)) {
+        // tslint:disable-next-line:no-string-literal
+        searchRequest.request.filters['phone'] = this.searchKeyword;
+    } else {
+        searchRequest.request.query = this.searchKeyword;
+    }
+
+    this.helperService.getAllUser(searchRequest).subscribe((response: any) => {
+        this.searchRes.searchStatus = 'end';
+        if (_.has(response, 'result.response.content')) {
+          this.searchRes.content = this.excludeCreatorAndCollaborators(_.get(response, 'result.response.content'));
+          if (this.searchRes.content.length) {
+            this.searchRes.isEmptyResponse = false;
+          } else {
+            this.searchRes.isEmptyResponse = true;
+            this.searchRes.content = [];
+          }
+        }
+    },
+    (error) => {
+      this.toasterService.error(_.get(this.configService, 'labelConfig.messages.error.001'));
+      console.log(error);
+      this.searchRes.content = [];
+      this.searchRes.isEmptyResponse = true;
+      this.searchRes.errorMessage = 'Oops! Something went wrong. Please try again later.';
```

Comment:
Why can't we use the same error message as  
```suggestion
      this.searchRes.errorMessage = _.get(this.configService, 'labelConfig.messages.error.001')
```